README.md

```diff
@@ -1,4 +1,4 @@
-# Container image source for ling-102/ling-194
+# Container image source for ling-102
 This is based on the previous [ling-194](https://github.com/UCSB-PSTAT/ling-194-docker-image) and [ling-100](https://github.com/UCSB-PSTAT/ling-110-docker-image) images.
```

Comment:
While we're here, ling-100 should be ling-101.  I won't block the PR over this, of course, but would be nice to clean up my typo..